templates/editEvents.html

```diff
@@ -164,17 +164,14 @@ <h3 class="panel-title">
                       <div id="modalReallyRemoveRound_{{eventCode}}" class="modal fade">
                         <div class="modal-dialog">
                           <div class="modal-content">
-                            <div class="modal-header">
-                              <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Cancel</span></button>
-                              <h4 class="modal-title">
-                                Are you sure you want to delete {{eventName eventCode}}: {{roundName lastRoundCode}}?
-                              </h4>
-                            </div>
+                            {{#cmHeader}}
+                              Are you sure you want to delete {{eventName eventCode}}: {{roundName lastRoundCode}}?
+                            {{/cmHeader}}
                             <div class="modal-body">
                               <p>It appears that this round contains results. If you delete it, those results will be lost.</p>
                             </div>
                             <div class="modal-footer">
-                              <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+                              {{> cmCancelButton}}
```

Comment:
OH! Gotcha. I'm fine with `ccmCancelButton` or `ccmModalCancelButton`. One crazy idea is to keep `cm` and rename `ccmModal` to `cm`. Whatever you think is best. As you said, it's easy to look these things up.